key.core/src/main/antlr4/KeYParser.g4

```diff
@@ -522,8 +538,9 @@ argument_list
     RPAREN
 ;
 
+integer_with_minux: MINUS? integer;
 integer:
-  (MINUS)? (INT_LITERAL | HEX_LITERAL | BIN_LITERAL)
```

Comment:
The minus is a prefix operator and already handled by `unary_minus_term`.

On the other side, there might be some rules, that want allow negative integers, e.g., in the non-expression part of the grammar.

Reply:
Yes.